gavel/models/item.py

```diff
@@ -33,16 +33,21 @@ class Item(db.Model):
     sigma_sq = db.Column(db.Float)
 
 
-    def __init__(self, name, location, description):
+    def __init__(self, name, location, description, desired_prizes):
         self.name = name
         self.location = location
         self.description = description
         self.mu = crowd_bt.MU_PRIOR
         self.sigma_sq = crowd_bt.SIGMA_SQ_PRIOR
+        
+        tracks_set = set(desired_prizes)
         # TODO Remove hard coding and add csv reading functionality
         tracks_data = {}
         for track in settings.TRACKS:
-            tracks_data[track] = 0
+            if track in tracks_set:
+                tracks_data[track] = 1
+            else:
+                tracks_data[track] = 0
```

Comment:
Once you change the desired_prizes to be just a set, you can do it as:
```
tracks_data[track] = track in desired_prizes
```
We don't really need an if for this, since all we're checking for is T/F.